recipes/azure-core-cpp/meta.yaml

```
@@ -0,0 +1,65 @@
+{% set version = "1.10.0" %}
+{% set sha256 = "e6ae26ad30376bef6272024564764ba7bf5756063dc94cc4b74180cd004ab75e" %}
+
+package:
+  name: azure-core-cpp
+  version: {{ version }}
+
+source:
+  url: https://github.com/Azure/azure-sdk-for-cpp/archive/refs/tags/azure-core_{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    # https://github.com/Azure/azure-sdk-for-cpp/issues/4784
+    # https://github.com/Azure/azure-sdk-for-cpp/pull/4785
+    - 0001-remove-wil-from-exported-config.patch
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage("azure-core-cpp", min_pin="x", max_pin="x") }}
+
+# https://github.com/Azure/azure-sdk-for-cpp/blob/main/sdk/core/azure-core/vcpkg.json
+requirements:
+  build:
+    - {{ compiler("cxx") }}
+    - cmake
+    - ninja
+  host:
+    - libcurl
+    - openssl  # [not win]
+    - wil  # [win]
+
+test:
+  commands:
+    # shared libs
+    - test -f ${PREFIX}/lib/libazure-core.so            # [linux]
+    - test -f ${PREFIX}/lib/libazure-core.dylib         # [osx]
```

Nit: this can be rewritten using and `SHLIB_EXT`. 
```suggestion
    - test -f ${PREFIX}/lib/libazure-core${SHLIB_EXT}   # [unix]
```

This suggestion also applies to the other libraries' recipe.

---

Applied. Thanks!